.golangci.yml

```diff
@@ -66,6 +66,10 @@ issues:
     - linters:
         - stylecheck
       text: "ST1003: struct field BrokerK8sApiServer"
+    - path: apis/submariner/v1alpha1/submariner_types.go
+      text: "ST1003: struct field NatEnabled should be NATEnabled"
+    - path: pkg/subctl/cmd/join.go
+      text: "ST1003: var disableNat should be disableNAT"
```

Comment:
`NATEnabled` is a Submariner spec variable. `disableNat` is a flag to `subctl join` and is deprecated so it would anyways be removed with https://github.com/submariner-io/submariner-operator/issues/1328

Reply:
right but the variables can be changed in code to comply:

NatEnabled  -> NATEnabled
disableNat -> disableNAT